content/shmem_alltoall.tex

```diff
@@ -103,7 +103,7 @@
     This routine restores \VAR{pSync} to its original contents.  Multiple calls
     to \openshmem\ routines that use the same \VAR{pSync} array do not require
     that \VAR{pSync} be reinitialized after the first call.
-    You must ensure that the \VAR{pSync} array is not being updated by any
+    The user must ensure that the \VAR{pSync} array is not being updated by any
```

Comment:
We should consider using a macro for "user". So we can get consistent formatting through out the document.